app/src/androidTest/java/com/sdp/movemeet/view/profile/EditProfileActivityTest.java

```diff
@@ -86,25 +98,25 @@ public void updateProfileByEnteringFieldsAndClickingSaveButton() {
 
     }
 
-    @Test
-    public void testAccessFirestoreUsersCollectionForUpdate() {
 
-        try (ActivityScenario<EditProfileActivity> scenario = ActivityScenario.launch(EditProfileActivity.class)) {
+    @After
+    public void signOut() {
```

Comment:
Why all the following part is in the @After ?

Reply:
This is because that's the last step of this test. Here I simply reset the user information as they were before the beginning of the test!

Reply:
Oh yeah ok it's simply to reset ! perfect !